app/index.js

```diff
@@ -94,7 +51,17 @@ var GeneratorGenerator = module.exports = generators.Base.extend({
       var prompts = [{
         name: 'githubUser',
         message: 'Would you mind telling me your username on GitHub?',
-        default: 'someuser'
+        default: function() {
+          var done = this.async();
+
+          self.user.github.username(function (err, username) {
```

Comment:
The issue is this code uses the `this.async()` from Inquirer. So there's a conflict about the `this`. Using self in this case is legitimate.